qa/standalone/ceph-helpers.sh

```diff
@@ -177,7 +177,8 @@ function teardown() {
       pattern=""
     fi
     # Local we start with core and teuthology ends with core
-    if ls $(dirname "$pattern") | grep -q '^core\|core$' ; then
+    # unittest cores can be lying around during parallel ctest, and are safe to ignore
```

Comment:
i don't think it's safe to ignore core.

> They are however not generated by the tests running with ceph-helpers.

but who else could generate these core ?

Reply:
@tchaikov
If you run `ctest -j 16` or something other in parallel, then there are others that might leave something behind.
The comment in the script suggest as much as that there is no real idea where to look for cores, so we just take the whole tree.
But one thins is certain: unittest cores are not generated by more complex scripts that use `ceph-helpers.sh`